### DISCUSSION TITLE
BOM validation

### DISCUSSION THREAD

**Discussion by StmLord** [frntc/RAD#4] [General]
Hi, I'm trying to build a board of RAD expansion, on the PCB there are places for 2x 74HCT30 but on the BOM only one is mentioned and marked on the boad sketch.
Is this a mistake on BOM or the second placement is not needed?
If its not needed then why is it there? what is the purpose?
Thanks for you amazing work on all of your projects!

**Comment by frntc** [comment]
Only the one marked is needed. The second one is a hardware-masking circuit for some signals, in case it's needed for something in the future :)
